packages/expo-calendar/src/Calendar.ts

```diff
@@ -112,11 +112,43 @@ export type Alarm = {
   method?: string; // Method, Android
 };
 
+export enum DayOfTheWeek {
+  Sunday = 1,
+  Monday,
+  Tuesday,
+  Wednesday,
+  Thursday,
+  Friday,
+  Saturday,
+}
```

Comment:
Please add values to all enum fields. In our other modules, we follow this rule. 

Should be:
```
export enum DayOfTheWeek {
  Sunday = 1,
  Monday = 2,
  Tuesday = 3,
  ...
}
```